src/Entity/Traits/EntityTrait.php

```
@@ -0,0 +1,49 @@
+<?php
+
+namespace Minormous\Dali\Entity\Traits;
+
+use Webmozart\Assert\Assert;
+use Minormous\Dali\Exceptions\InvalidEntityException;
+
+/**
+ * @psalm-require-implements \Minormous\Dali\Entity\Interfaces\EntityInterface
+ */
+trait EntityTrait
+{
+    use ToArray;
+
+    public function __construct(array $values)
+    {
+        foreach ($values as $key => $value) {
+            if (!\property_exists($this, $key)) {
+                continue;
+            }
```

should we throw on this? Would expose invalid code easily.